bitrise.yml

```diff
@@ -31,6 +31,31 @@ workflows:
         title: Step Test
         inputs:
         - target: emulator
+    - script:
+        title: Output test
+        inputs:
+        - content: |
+            #!/usr/bin/env bash
+            if [[ ! -e "$BITRISE_DSYM_DIR_PATH" ]]; then
+                echo "Does not exist: dSYM dir's path"
+                exit 1
+            fi
```

Comment:
Instead of
```bash
if [[ -e "..." ]]; then
  echo "..."
fi
if [[ -e "..." ]]; then
  echo "..."
fi
```
shorter to use
```bash
if [[ -e "..." ]]; then
  echo "..."
elif [[ -e "..." ]]; then
  echo "..."
fi
```